### PR TITLE
Update 1Password CLI download & munki recipes

### PR DESCRIPTION
**Pull request by apizz**
- Fix spacing issues
- Simplify regex in download recipe and remove no longer needed URL input variable
- Add CodeSignatureVerifier for both checks on the signed PKG and signed CLI binary
- Use clean version in munki recipe rather than PKG default `v%version%`

Verbose output from successful munki recipe run:

```
autopkg run -vv 1Password_CLI.munki.recipe                                                                                 00:07:18
Processing 1Password_CLI.munki.recipe...
WARNING: 1Password_CLI.munki.recipe is missing trust info and FAIL_RECIPES_WITHOUT_TRUST_INFO is not set. Proceeding...
URLTextSearcher
{'Input': {'re_pattern': '(https://cache.agilebits.com/dist/1P/op/pkg/v([\\d\\.]+)/op_darwin_amd64_v(?P<version>[\\d\\.]+).pkg)',
           'url': 'https://app-updates.agilebits.com/product_history/CLI'}}
URLTextSearcher: No value supplied for result_output_var_name, setting default value of: match
URLTextSearcher: Found matching text (version): 1.8.0
URLTextSearcher: Found matching text (match): https://cache.agilebits.com/dist/1P/op/pkg/v1.8.0/op_darwin_amd64_v1.8.0.pkg
{'Output': {'match': 'https://cache.agilebits.com/dist/1P/op/pkg/v1.8.0/op_darwin_amd64_v1.8.0.pkg',
            'version': '1.8.0'}}
URLDownloader
{'Input': {'filename': '1Password CLI.pkg',
           'url': 'https://cache.agilebits.com/dist/1P/op/pkg/v1.8.0/op_darwin_amd64_v1.8.0.pkg'}}
URLDownloader: No value supplied for prefetch_filename, setting default value of: False
URLDownloader: No value supplied for CHECK_FILESIZE_ONLY, setting default value of: False
URLDownloader: Storing new Last-Modified header: Thu, 22 Oct 2020 18:07:38 GMT
URLDownloader: Storing new ETag header: "5f91ca6a-49b2fe"
URLDownloader: Downloaded /Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/downloads/1Password CLI.pkg
{'Output': {'download_changed': True,
            'etag': '"5f91ca6a-49b2fe"',
            'last_modified': 'Thu, 22 Oct 2020 18:07:38 GMT',
            'pathname': '/Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/downloads/1Password '
                        'CLI.pkg',
            'url_downloader_summary_result': {'data': {'download_path': '/Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/downloads/1Password '
                                                                        'CLI.pkg'},
                                              'summary_text': 'The following '
                                                              'new items were '
                                                              'downloaded:'}}}
EndOfCheckPhase
{'Input': {}}
{'Output': {}}
CodeSignatureVerifier
{'Input': {'expected_authority_names': ['Developer ID Installer: AgileBits '
                                        'Inc. (2BUA8C4S2C)',
                                        'Developer ID Certification Authority',
                                        'Apple Root CA'],
           'input_path': '/Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/downloads/1Password '
                         'CLI.pkg'}}
CodeSignatureVerifier: Verifying installer package signature...
CodeSignatureVerifier: Package "1Password CLI.pkg":
CodeSignatureVerifier:    Status: signed by a developer certificate issued by Apple for distribution
CodeSignatureVerifier:    Signed with a trusted timestamp on: 2020-10-22 18:04:49 +0000
CodeSignatureVerifier:    Certificate Chain:
CodeSignatureVerifier:     1. Developer ID Installer: AgileBits Inc. (2BUA8C4S2C)
CodeSignatureVerifier:        Expires: 2024-10-23 17:10:43 +0000
CodeSignatureVerifier:        SHA256 Fingerprint:
CodeSignatureVerifier:            14 1D D8 7B 2B 23 12 11 F1 44 08 49 79 80 07 DF 62 1D E6 EB 3D AB 
CodeSignatureVerifier:            98 5B C9 64 EE 97 04 C4 A1 C1
CodeSignatureVerifier:        ------------------------------------------------------------------------
CodeSignatureVerifier:     2. Developer ID Certification Authority
CodeSignatureVerifier:        Expires: 2027-02-01 22:12:15 +0000
CodeSignatureVerifier:        SHA256 Fingerprint:
CodeSignatureVerifier:            7A FC 9D 01 A6 2F 03 A2 DE 96 37 93 6D 4A FE 68 09 0D 2D E1 8D 03 
CodeSignatureVerifier:            F2 9C 88 CF B0 B1 BA 63 58 7F
CodeSignatureVerifier:        ------------------------------------------------------------------------
CodeSignatureVerifier:     3. Apple Root CA
CodeSignatureVerifier:        Expires: 2035-02-09 21:40:36 +0000
CodeSignatureVerifier:        SHA256 Fingerprint:
CodeSignatureVerifier:            B0 B1 73 0E CB C7 FF 45 05 14 2C 49 F1 29 5E 6E DA 6B CA ED 7E 2C 
CodeSignatureVerifier:            68 C5 BE 91 B5 A1 10 01 F0 24
CodeSignatureVerifier: 
CodeSignatureVerifier: Signature is valid
CodeSignatureVerifier: Authority name chain is valid
{'Output': {}}
FlatPkgUnpacker
{'Input': {'destination_path': '/Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/unpack',
           'flat_pkg_path': '/Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/downloads/1Password '
                            'CLI.pkg'}}
FlatPkgUnpacker: Unpacked /Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/downloads/1Password CLI.pkg to /Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/unpack
{'Output': {}}
PkgPayloadUnpacker
{'Input': {'destination_path': '/Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/unpack',
           'pkg_payload_path': '/Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/unpack/Payload'}}
PkgPayloadUnpacker: Unpacked /Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/unpack/Payload to /Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/unpack
{'Output': {}}
CodeSignatureVerifier
{'Input': {'expected_authority_names': [],
           'input_path': '/Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/unpack/op',
           'requirement': 'identifier "com.1password.op" and anchor apple '
                          'generic and certificate '
                          '1[field.1.2.840.113635.100.6.2.6] /* exists */ and '
                          'certificate leaf[field.1.2.840.113635.100.6.1.13] '
                          '/* exists */ and certificate leaf[subject.OU] = '
                          '"2BUA8C4S2C"',
           'strict_verification': True}}
CodeSignatureVerifier: Verifying code signature...
CodeSignatureVerifier: Deep verification enabled...
CodeSignatureVerifier: Strict verification enabled...
CodeSignatureVerifier: /Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/unpack/op: valid on disk
CodeSignatureVerifier: /Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/unpack/op: satisfies its Designated Requirement
CodeSignatureVerifier: /Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/unpack/op: explicit requirement satisfied
CodeSignatureVerifier: Signature is valid
{'Output': {}}
PathDeleter
{'Input': {'path_list': ['/Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/unpack']}}
PathDeleter: Deleted /Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/unpack
{'Output': {}}
MunkiPkginfoMerger
{'Input': {'additional_pkginfo': {'version': '1.8.0'},
           'pkginfo': {'catalogs': ['testing'],
                       'category': 'Utilities',
                       'description': 'The 1Password command-line tool makes '
                                      'your 1Password account accessible '
                                      'entirely from the command line. A '
                                      'simple op signin will securely '
                                      'authenticate you with the 1Password '
                                      'service and give you access to a wide '
                                      'range of capabilities.',
                       'developer': 'Agilebits',
                       'display_name': '1Password Command Line Interface',
                       'name': '1Password CLI',
                       'unattended_install': True}}}
MunkiPkginfoMerger: Merged {'version': '1.8.0'} into pkginfo
{'Output': {'pkginfo': {'catalogs': ['testing'],
                        'category': 'Utilities',
                        'description': 'The 1Password command-line tool makes '
                                       'your 1Password account accessible '
                                       'entirely from the command line. A '
                                       'simple op signin will securely '
                                       'authenticate you with the 1Password '
                                       'service and give you access to a wide '
                                       'range of capabilities.',
                        'developer': 'Agilebits',
                        'display_name': '1Password Command Line Interface',
                        'name': '1Password CLI',
                        'unattended_install': True,
                        'version': '1.8.0'}}}
MunkiImporter
{'Input': {'MUNKI_REPO': '/Users/Shared/munki_repo',
           'pkg_path': '/Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/downloads/1Password '
                       'CLI.pkg',
           'pkginfo': {'catalogs': ['testing'],
                       'category': 'Utilities',
                       'description': 'The 1Password command-line tool makes '
                                      'your 1Password account accessible '
                                      'entirely from the command line. A '
                                      'simple op signin will securely '
                                      'authenticate you with the 1Password '
                                      'service and give you access to a wide '
                                      'range of capabilities.',
                       'developer': 'Agilebits',
                       'display_name': '1Password Command Line Interface',
                       'name': '1Password CLI',
                       'unattended_install': True,
                       'version': '1.8.0'},
           'repo_subdirectory': 'apps/1Password CLI'}}
MunkiImporter: No value supplied for MUNKI_REPO_PLUGIN, setting default value of: FileRepo
MunkiImporter: No value supplied for MUNKILIB_DIR, setting default value of: /usr/local/munki
MunkiImporter: No value supplied for force_munki_repo_lib, setting default value of: False
MunkiImporter: Using repo lib: AutoPkgLib
MunkiImporter:         plugin: FileRepo
MunkiImporter:           repo: /Users/Shared/munki_repo
MunkiImporter: Copied pkginfo to: /Users/Shared/munki_repo/pkgsinfo/apps/1Password CLI/1Password CLI-1.8.0.plist
MunkiImporter:            pkg to: /Users/Shared/munki_repo/pkgs/apps/1Password CLI/1Password CLI-1.8.0.pkg
{'Output': {'munki_importer_summary_result': {'data': {'catalogs': 'testing',
                                                       'icon_repo_path': '',
                                                       'name': '1Password CLI',
                                                       'pkg_repo_path': 'apps/1Password '
                                                                        'CLI/1Password '
                                                                        'CLI-1.8.0.pkg',
                                                       'pkginfo_path': 'apps/1Password '
                                                                       'CLI/1Password '
                                                                       'CLI-1.8.0.plist',
                                                       'version': '1.8.0'},
                                              'report_fields': ['name',
                                                                'version',
                                                                'catalogs',
                                                                'pkginfo_path',
                                                                'pkg_repo_path',
                                                                'icon_repo_path'],
                                              'summary_text': 'The following '
                                                              'new items were '
                                                              'imported into '
                                                              'Munki:'},
            'munki_info': {'_metadata': {'created_by': 'autopkgadmin',
                                         'creation_date': datetime.datetime(2021, 3, 24, 4, 9, 22),
                                         'munki_version': '5.2.2.4286',
                                         'os_version': '10.15.7'},
                           'autoremove': False,
                           'catalogs': ['testing'],
                           'category': 'Utilities',
                           'description': 'The 1Password command-line tool '
                                          'makes your 1Password account '
                                          'accessible entirely from the '
                                          'command line. A simple op signin '
                                          'will securely authenticate you with '
                                          'the 1Password service and give you '
                                          'access to a wide range of '
                                          'capabilities.',
                           'developer': 'Agilebits',
                           'display_name': '1Password Command Line Interface',
                           'installed_size': 13189,
                           'installer_item_hash': '6decb453be4e507336b004a0169befcee7dc9918198ae6fa33d0b55d8e8acc5f',
                           'installer_item_location': 'apps/1Password '
                                                      'CLI/1Password '
                                                      'CLI-1.8.0.pkg',
                           'installer_item_size': 4716,
                           'minimum_os_version': '10.5.0',
                           'name': '1Password CLI',
                           'receipts': [{'installed_size': 13189,
                                         'packageid': 'com.1password.op',
                                         'version': 'v1.8.0'}],
                           'unattended_install': True,
                           'uninstall_method': 'removepackages',
                           'uninstallable': True,
                           'version': '1.8.0'},
            'munki_repo_changed': True,
            'pkg_repo_path': '/Users/Shared/munki_repo/pkgs/apps/1Password '
                             'CLI/1Password CLI-1.8.0.pkg',
            'pkginfo_repo_path': '/Users/Shared/munki_repo/pkgsinfo/apps/1Password '
                                 'CLI/1Password CLI-1.8.0.plist'}}
Receipt written to /Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/receipts/1Password_CLI.munki-receipt-20210324-000922.plist

The following new items were downloaded:
    Download Path                                                                                          
    -------------                                                                                          
    /Users/autopkgadmin/Library/AutoPkg/Cache/com.github.apettinen.munki.1Password_CLI/downloads/1Password CLI.pkg  

The following new items were imported into Munki:
    Name           Version  Catalogs  Pkginfo Path                                  Pkg Repo Path                               Icon Repo Path  
    ----           -------  --------  ------------                                  -------------                               --------------  
    1Password CLI  1.8.0    testing   apps/1Password CLI/1Password CLI-1.8.0.plist  apps/1Password CLI/1Password CLI-1.8.0.pkg 
```

### CODE REVIEW

File: 1Password_CLI/1Password_CLI.download.recipe [hunk 1]
```diff
@@ -13,41 +13,102 @@
...
+        </dict>
+        <dict>
+            <key>Arguments</key>
+            <dict>
+                <key>path_list</key>
+                <array>
+                    <string>%RECIPE_CACHE_DIR%/unpack</string>
+                </array>
+            </dict>
+            <key>Processor</key>
```
[the remark this answers is not in the document]
Comment (apizz) [reply 1Password_CLI/1Password_CLI.download.recipe]: Good call out.  I'm usually good about following folks organization scheme RE this and using spaces / tabs.  Should be good now.

**Comment by apizz** [comment]
Should be good now. LMK if you want any additional changes.

**Review by apettinen — APPROVED** [review]
Looks good, thanks!

**Comment by apettinen** [comment]
Thanks for your effort, good stuff! 👍

**Pull request merged**